charts/cachemachine/templates/clusterrole.yml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRole
```

Comment:
Ideally the `daemonsets` stuff would be in a `Role` rather than a `ClusterRole` to keep it from having permissions to delete random other `DaemonSets`, since if I'm not mistaken it only needs to create and delete `DaemonSets` in its own namespace.

Reply:
Good call, will do